Firestore/Source/API/FSTUserDataConverter.h

```diff
@@ -88,11 +88,12 @@ NS_ASSUME_NONNULL_BEGIN
 - (instancetype)init NS_UNAVAILABLE;
 
 - (instancetype)initWithKey:(FSTDocumentKey *)key
-                 databaseID:(firebase::firestore::model::DatabaseId)databaseID
+                 databaseID:(const firebase::firestore::model::DatabaseId *)databaseID
     NS_DESIGNATED_INITIALIZER;
 
 @property(nonatomic, strong, readonly) FSTDocumentKey *key;
-@property(nonatomic, assign, readonly) firebase::firestore::model::DatabaseId databaseID;
+// Does not own the DatabaseId instance.
```

Comment:
I think that our general rule should be that raw pointers are non-owning.

If that's the case then this comment becomes unnecessary, no?

Reply:
Comment is cheap; I'd rather make this clear than cause possible misunderstanding and bugs. Sometimes, ownership is passed over and thus raw pointers may take ownership as well.